### ISSUE TITLE
New switch doesn't get assign rules

### ISSUE THREAD

**Issue by capstan1** [dresden-elektronik/deconz-rest-plugin#1538]
Short story: I was configuring my system like this: added all switches and lights to deconz and then configured the switches with Hue Essentials app because that gave me the possibility of controling various light groups with one switch. Now I read that if you configure switches from Phoscon by adding lights to a Switch group the switch will even work if the RaspBee is off / if the Raspberry is frozen.
So I deleted a switch, re-added it and assigned lights to it. But the only button that works on my philips dimmer is the off button. Also a Ikea dimmer doesn't work. 

I looked into the Hue Essentials App and it seems like there are no rules written to the switch. Only action I found is "Off" on "short release" of the off button. The other 3 buttons don't have any rules.
If I add a new rule via Hue Essentials to turn lights on, it will work, though. So it seems that the configuration GUI of phoscon doesn't apply the rules needed.

**TL;DR: configuring a new switch with phoscon doesn't add rules to the buttons**

Hardware: RaspBee on a Raspberry B 3B+
connected to HomeAssistant using marthoc Addon
Firmware: 26320500
